testing/functional_tests_assertions.rst

```diff
@@ -59,8 +69,9 @@ Crawler
 
 .. note::
 
-    You cannot use the :doc:`Crawler </components/dom_crawler>` assertions,
-    when using `symfony/panther`_ for end-to-end testing.
+    The following assertions only work if a request has been made with the
+    ``Client`` in a test case extending the ``WebTestCase`` class. They are not
+    available either when using `symfony/panther`_ for end-to-end testing.
```

Comment:
Sounds weird to use `either` without a following `or`. A left over? I think we should remove it and just say "not available when".